### DISCUSSION TITLE
How can I add handlers to controls in (context) flyout?

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#14657] [Q&A] [answered]
I have a context flyout  associated with a TreeView.
I can access the flyout with TreeView.ContextFlyout.

How can I access the contents of the flyout so I can add e.g. keydown handlers?

**Comment by timunie** [comment]
Sounds like a good task for a custom behavior to me 🤔

**Reply by wgraaf** [reply]
Is there an example on how to go about this?
It would also work if I can access the flyout contents in code somehow, which is not obvious from the properties/methods exposed by the flyout.

**Comment by maxkatz6** [comment, marked as answer]
> How can I access the contents of the flyout so I can add e.g. keydown handlers?

Cast `Flyout.Content`? Otherwise, it's not really clear what's your problem is.
